JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -69,19 +60,24 @@ ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
 	results[[".meta"]] <- meta
 
 	errorList <- NULL
-
-	if (is.null(resultsAlpha) && !is.null(variables) && length(variables) > 0) {
-
-		# check for errors
-		anyErrors <- .hasErrors(dataset = dataset, perform = perform,
-								type = c("infinity", "variance", "observations"),
-								observations.amount = " < 3",
-								exitAnalysisIfErrors = TRUE)
-
-		resultsAlpha <- .reliabilityResults(dataset, options, variables, perform)
-
-	} # otherwise results retrieved from state
-
+	
+	if (!is.null(variables) && length(variables) > 0) {
+		if (is.null(resultsAlpha)) { # was the main analysis retrieved from state?
```

Comment:
That would break the analysis, but it should not be possible because:


```
        # lines 43-49
	defaults = c("variables", "reverseScaledItems", "missingValues")
	stateKey <- list(
		resultsAlpha = defaults,
		confAlpha = c(defaults, "confAlphaLevel")
	)
	resultsAlpha <- state[["resultsAlpha"]]
	resultsAlpha[["ciAlpha"]] <- state[["confAlpha"]]
```

should ensure that if `resultsAlpha ` is not retrieved from the state then neither is `confAlpha`.